stdlib/Makefile

```
@@ -264,6 +264,7 @@ depend:
 	  > .depend.tmp
 	$(CAMLDEP) $(DEPFLAGS) -pp "$(AWK) -f ./remove_module_aliases.awk" \
 	  stdlib.ml stdlib.mli >> .depend.tmp
+	$(strings $(which sed) | grep -q 'GNU') || (echo "the sed in PATH is not GNU sed" && exit 1)
```

```suggestion
	$$(grep -q 'GNU' $(which sed)) || {echo "the sed in PATH is not GNU sed" && exit 1}
```

Prefer `{` `}` over `(` `)`
Avoid `|` and dependencies on `strings` and directly call `grep`
Also you need to escape `$` with `$$` or use `$(shell)`

---

thx, sorry